domain/services/model.go

```diff
@@ -347,7 +344,6 @@ func (s *ModelServices) Annotation() *annotationService.Service {
 func (s *ModelServices) Storage() *storageservice.Service {
```

Comment:
Probably prefer to keep this. The first time we've got an issue reported, and someone's just going to put it back in order to add diagnostic logging.